.devops/deploy-pipelines.yml

```diff
@@ -0,0 +1,157 @@
+# Azure DevOps pipeline to release a new version and deploy to production.
+
+variables:
+  NODE_VERSION: '8.9.4'
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+
+parameters:
+  - name: 'RELEASE_SEMVER'
+    displayName: 'When packing a release, define the version bump to apply'
+    type: string
+    values:
+      - major
+      - minor
+      - patch
+    default: minor
+
+  - name: 'TEST_ENABLE_DEPLOY'
+    displayName: 'Enable deploy in test environment'
+    type: boolean
+    default: false
+
+  - name: 'TEST_DEPLOY_TYPE'
+    displayName: 'Method to achieve deployment in Test (if enabled):'
+    type: string
+    default: deployToTestSlot
+    values:
+      - deployToTestSlot  
+      - deployToProductionSlot
+
+  - name: 'PRODUCTION_ENABLE_DEPLOY'
+    displayName: 'Enable deploy in production environment'
+    type: boolean
+    default: true
+
+  - name: 'PRODUCTION_DEPLOY_TYPE'
+    displayName: 'Method to achieve deployment in Production (if enabled):'
+    type: string
+    default: deployToProductionSlot
+    values:
+      - deployToStagingSlot  
+      - deployToProductionSlot
+      - deployToStagingSlotAndSwap
```

Comment:
I'll keep it simple: just deploy in prod and staging without checks.

If in need of something different, we can disable a specific stage from the GUI when running the pipeline

Reply:
@balanza ok, but how can we test the deploy without a staging env and a test env?

Reply:
done [here](https://github.com/pagopa/io-pagopa-proxy/pull/293/commits/40c89578d435c5c0731ab2cf721252c8bf2f2ffc)

Reply:
> but how can we test the deploy without a staging env and a test env?

The actual doesn't include a staging step, so staging env is pretty useless. The only scenario is when we really want to try a version on a sandbox environment. Do do that, we can disable the production stage right before running the pipeline.